virttest/libvirt_vm.py

```diff
@@ -986,6 +986,8 @@ def add_kernel(
                     result += "initrd=%s," % initrd_path
                 if has_sub_option("boot", "kernel_args") and kernel_args:
                     result += 'kernel_args="%s",' % kernel_args
```

Comment:
Now that kernel_args is not part of virt_install shall we remove the check.

Reply:
@harihare 
I kept that for people still using the older virt-install tool which has kernel_args parameter listed in `virt-install --boot help`
If that is not required, then I can remove that check